lib/blocks.php

```diff
@@ -45,6 +45,7 @@ function gutenberg_reregister_core_block_types() {
 				'text-columns',
 				'verse',
 				'video',
+				'embed',
```

Comment:
Is this needed?

Reply:
Ah, I suppose this is addressing this other piece of feedback? https://github.com/WordPress/gutenberg/pull/24090#issuecomment-708950360

Reply:
Actually is not related and is about the above feedback. If this PR will not land, I'll create a separate one to address that.